### DISCUSSION TITLE
Feature: trigger GitHub Action on commit comments

### DISCUSSION THREAD

**Discussion by mpareja** [community/community#61934] [Actions]
### Select Topic Area

Product Feedback

### Body

The events that [trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#about-events-that-trigger-workflows) include `issue_comment` and `pull_request_review_comment` but does not include `commit_comment`.

**Comment by kgryte** [comment]
For our repository maintenance use cases, we'd love to be able to trigger workflows from commit comments. This would allow us to issue bot commands for creating follow-up tasks and create maintenance issues.
